pyproject.toml

```diff
@@ -90,6 +90,7 @@ xfail_strict = true
 empty_parameter_set_mark = "fail_at_collect"
 addopts = "--strict-markers"
 markers = [
+    "fast: mark automatically applied to tests which don't have the \"slow\" mark applied (deselect with '-m \"not fast\"')",
```

Comment:
I'm still fuzzy on how this works? If I want to run fast I do `-m fast "not slow"` and vise versa `-m slow "not fast"`?

Reply:
`-m fast` or `-m "not slow"`. Same applies for slow/not fast